### PR TITLE
Add Postgres dockerfile

### PR DESCRIPTION
**Pull request by ttung**
### Description
This is pulled from https://github.com/docker-library/postgres/commit/e8cb2acbb372f0c89219792f82b0bdd2e405602f

### Test plan
Used in subsequent PR

### CODE REVIEW

File: docker/postgres/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,182 @@
...
+	; \
+	rm postgresql.tar.bz2; \
+	\
+	apk add --no-cache --virtual .build-deps \
+		bison \
+		coreutils \
+		dpkg-dev dpkg \
+		flex \
+		gcc \
+#		krb5-dev \
```
Comment (phoenixAja) [inline docker/postgres/Dockerfile:43]: should keep this?

File: docker/postgres/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,182 @@
...
+		libc-dev \
+		libedit-dev \
+		libxml2-dev \
+		libxslt-dev \
+		linux-headers \
+		llvm10-dev clang g++ \
+		make \
+#		openldap-dev \
```
Comment (phoenixAja) [inline docker/postgres/Dockerfile:51]: same

File: docker/postgres/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,182 @@
...
+		openssl-dev \
+# configure: error: prove not found
+		perl-utils \
+# configure: error: Perl module IPC::Run is required to run TAP tests
+		perl-ipc-run \
+#		perl-dev \
+#		python-dev \
+#		python3-dev \
+#		tcl-dev \
```
Comment (phoenixAja) [inline docker/postgres/Dockerfile:60]: same

File: docker/postgres/Dockerfile [hunk 4]
```diff
@@ -0,0 +1,182 @@
...
+			| tr ',' '\n' \
+			| sort -u \
+			| awk 'system("[ -e /usr/local/lib/" $1 " ]") == 0 { next } { print "so:" $1 }' \
+	)"; \
+	apk add --no-cache --virtual .postgresql-rundeps \
+		$runDeps \
+		bash \
+		su-exec \
+# tzdata is optional, but only adds around 1Mb to image size and is recommended by Django documentation:
+# https://docs.djangoproject.com/en/1.10/ref/databases/#optimizing-postgresql-s-configuration
```
Comment (phoenixAja) [inline docker/postgres/Dockerfile:125]: is it also recommended by flask?

**Comment by ttung** [comment]
I'm just importing their dockerfile verbatim.  I only plan on making the minimal set of changes to get what I want in #135.

File: docker/postgres/docker-entrypoint.sh [hunk 5]
```diff
@@ -0,0 +1,2482 @@
+
```
Comment (phoenixAja) [inline docker/postgres/docker-entrypoint.sh:1]: is this the right file? It looks like it's all html

File: docker/postgres/Dockerfile [hunk 6]
```diff
@@ -0,0 +1,182 @@
...
+STOPSIGNAL SIGINT
+#
+# An additional setting that is recommended for all users regardless of this
+# value is the runtime "--stop-timeout" (or your orchestrator/runtime's
+# equivalent) for controlling how long to wait between sending the defined
+# STOPSIGNAL and sending SIGKILL (which is likely to cause data corruption).
+#
+# The default in most runtimes (such as Docker) is 10 seconds, and the
+# documentation at https://www.postgresql.org/docs/12/server-start.html notes
+# that even 90 seconds may not be long enough in many instances.
```
Comment (phoenixAja) [inline docker/postgres/Dockerfile:179]: should we add this setting?

**Comment by phoenixAja** [comment]
> I'm just importing their dockerfile verbatim. I only plan on making the minimal set of changes to get what I want in #135.

ok i'll just approve then lol

**Review by phoenixAja — APPROVED** [review]

**Pull request merged**